roles/flannel/tasks/main.yml

```diff
@@ -51,3 +51,11 @@
   command: /sbin/iptables -t nat -A POSTROUTING -o {{ flannel_interface }}  -j MASQUERADE -m comment --comment "Allow external network access"
   notify:
     - save iptable rules
+
+- name: Allow DNS access
+  command: /sbin/iptables -A OS_FIREWALL_ALLOW -p {{ item }} -m state --state NEW -m {{ item }} --dport 53 -j ACCEPT -m comment --comment "Allow DNS {{ item }} access"
```

Comment:
I'm not familiar with this chain off the top of my head.

Is there a drop as the last rule?  If so, we need to use -I here instead of -A.

This looks like an inbound rule.  We probably don't need to specify --state New here.

Reply:
can't we use the `os_firewall_manage_iptables` rather than the `command` module?

Reply:
>Is there a drop as the last rule? If so, we need to use -I here instead of -A.

There is no DROP there - https://gist.github.com/vrutkovs/0aafa0889810a2fa297793f1a6dabd06

>This looks like an inbound rule. We probably don't need to specify --state New here.

Right, this is not needed, verified on a 3 master setup - pods reach DNS

>can't we use the `os_firewall_manage_iptables`

`os_firewall_manage_iptables` is a var, not a module